docker/Dockerfile

```diff
@@ -1,6 +1,15 @@
 FROM ubuntu:14.04
 MAINTAINER svanoort <samvanoort@gmail.com>
 
+# User id of the build user
+ARG uid=4242
```

Comment:
@olivergondza I think maybe jenkins & 1000 would make more sense for general users building jenkins packages, and then pass args for that specific builder?